src/LinAlg/hiopVectorRajaPar.cpp

```diff
@@ -1970,7 +1970,12 @@ bool hiopVectorRajaPar::isfinite_local() const
  */
 void hiopVectorRajaPar::print(FILE* file, const char* msg/*=NULL*/, int max_elems/*=-1*/, int rank/*=-1*/) const
```

Comment:
maybe be explicit that some arguments have default values; also replace NULL with nullptr

Reply:
Done!